__tests__/RouterView.spec.ts

```diff
@@ -308,40 +311,44 @@ describe('RouterView', () => {
   })
 
   describe('warnings', () => {
-    it('does not warn RouterView is wrapped', async () => {
+    it('does not warn RouterView is wrapped', () => {
       const route = createMockedRoute(routes.root)
-      const wrapper = await mount(
+      const wrapper = mount(
         {
           template: `
         <div>
           <router-view/>
         </div>
         `,
-        },
+        } as any,
```

Comment:
Is the `as any` necessary? I should be okay without it

Reply:
I removed it.